mobile/src/main/java/ch/epfl/sdp/mobile/application/social/SocialFacade.kt

```diff
@@ -28,4 +30,20 @@ class SocialFacade(private val auth: Auth, private val store: Store) {
       it.mapNotNull { doc -> doc?.toProfile() }
     }
   }
+
+  /**
+   * Get user by name
+   *
+   * @param text search criteria.
+   */
+  fun get(name: String): Flow<Profile> {
```

Comment:
[Nitpick] By our documentation convention, you could use the past tense for the function description and add the variable name next to the "@param"